client/my-sites/pages/blog-posts-page/style.scss

```diff
@@ -22,6 +22,7 @@
 	color: $gray-dark;
 	font-weight: bold;
 	font-family: $serif;
+	font-size: 18px;
 	margin-right: 33px;
```

Comment:
Adding the [`clear-text` mixin](https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/assets/stylesheets/shared/mixins/_clear-text.scss#L5) (or just declaring those rules explicitly) will even out the "boldness" discrepancy between the two fonts.